### DISCUSSION TITLE
Flag translations as requiring update when change source string

### DISCUSSION THREAD

**Discussion by staffordsmith83** [WeblateOrg/weblate#8870] [Q&A]
We need to change a source string in our application, and would like to flag this in Weblate as requiring re-translation.
How do we do this?

For Example:
We are making a simple change, from "Raster Map Quality" to "Raster Quality" - but the French translation still reads "Qualité de la carte raster"

**Comment by nijel** [comment]
Such string should be marked as "Needs editing" in Weblate after the source string has been changed.
